tests/test_cli.py

```diff
@@ -270,7 +270,7 @@ def test_local_run(mock_docker_installation):
     """Test that the run command works with different arguments."""
     args = argparse.Namespace(
         local=True,
-        ref="some/path",
+        ref=__name__,
```

Comment:
Why are we using `__name__` here?

Reply:
I don't understand completely why the tests have worked before. The actual module contains a test pipeline which is used as reference for the test cases. `__name__` makes more sense here.
However, I would cleanup the test cases as well. I think we can extract some fixtures here and make it more robust.